tree/ntuple/v7/src/RField.cxx

```diff
@@ -2070,6 +2088,11 @@ std::uint32_t ROOT::Experimental::RUnsplitField::GetTypeVersion() const
    return fClass->GetClassVersion();
 }
 
+std::uint32_t ROOT::Experimental::RUnsplitField::GetStreamerChecksum() const
+{
+   return fClass->GetCheckSum();
```

Comment:
We're sometimes using `Checksum` and sometimes `CheckSum`: can we uniform our nomenclature?
I would opt for always using `Checksum`.

Reply:
In principle I agree. The `CheckSum` is preexisting from `TClass`, so we cannot change that. But the new interfaces, I think, should use `Checksum`...

Reply:
Ah, right, I didn't notice it's not in our own API. Sorry for the noise